src/main/java/duke/command/TodoCommand.java

```diff
@@ -0,0 +1,36 @@
+package duke.command;
+
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+import duke.ui.Ui;
+import duke.storage.Storage;
+
+/**
+ * Represents a call to create a new Todo Task.
+ */
+public class TodoCommand extends Command {
+    public static final String COMMAND_WORD = "todo";
```

Comment:
Could leave a blank line here to follow coding standard.

Reply:
Blank line between line 12-13?